similarity/config/datastore.properties

```diff
@@ -0,0 +1,3 @@
+ENDPOINT	http://smaug.cs.illinois.edu:8080
```

Comment:
Drop this file. You should use this Resource configurator: 
https://github.com/CogComp/cogcomp-nlp/blob/365172b5b64c70430e248dd692e823c66645cde1/core-utilities/src/main/java/edu/illinois/cs/cogcomp/core/resources/ResourceConfigurator.java#L18